tests/data/models/Omni/model.yaml

```diff
@@ -724,6 +724,77 @@ views:
       description: null
   lastUpdatedTime: 1729345477397
   createdTime: 1729345477397
+- space: sp_pygen_models
```

Comment:
Downloaded the response version of the view which is used as a basis for the examples SDKs.